src/apps/explorer/ExplorerApp.tsx

```diff
@@ -25,11 +25,14 @@ import { version } from '../../../package.json'
 const SENTRY_DSN = process.env.REACT_APP_SENTRY_DSN
 const SENTRY_TRACES_SAMPLE_RATE = process.env.REACT_APP_SENTRY_TRACES_SAMPLE_RATE
 
+console.log('React app sentry dsn', SENTRY_DSN)
+console.log('GOOGLE_ANALYTICS_ID', process.env.GOOGLE_ANALYTICS_ID)
```

Comment:
Do we want to log these? Happy to do so if we really want. just asking

Reply:
No this was just for debugging why there are no sentry issues for gp-ui. This will be removed.